Source/SIMPLib/CoreFilters/util/ABSOperator.cpp

```diff
@@ -58,7 +58,11 @@ ABSOperator::~ABSOperator() = default;
 // -----------------------------------------------------------------------------
 //
 // -----------------------------------------------------------------------------
-void ABSOperator::calculate(AbstractFilter* filter, DataArrayPath calculatedArrayPath, QStack<ICalculatorArray::Pointer>& executionStack)
+void ABSOperator::calculate(AbstractFilter* filter, DataArrayPath calculatedArrayPath,
+                            QStack<ICalculatorArray::Pointer>& executionStack){CREATE_NEW_ARRAY_STANDARD_UNARY(filter, calculatedArrayPath, executionStack, fabs)}
```

Comment:
I don't think clang-format did this, but if it did, something needs to be tweaked.  I more than half-expected this line to be a lambda and was wondering where the `[=]` and closing parenthesis were.

Reply:
This seems to be a recurring problem in ArrayCalculator's operators for this same method.